packages/react-reconciler/src/__tests__/ReactCache-test.js

```diff
@@ -739,14 +739,20 @@ describe('ReactCache', () => {
     await act(async () => {
       refresh();
     });
-    expect(Scheduler).toHaveYielded(['Cache miss! [A]', 'Loading...']);
+    expect(Scheduler).toHaveYielded([
+      'Cache miss! [A]',
+      'Loading...',
+      // TODO: This happens too early, because we don't retain the refreshed
+      // cache until it commits. Will fix in next step.
+      'Cache cleanup: A [v1]',
```

Comment:
This explanation isn't right, but I believe I know what the issue is. Going offline for a bit. I'll resume this on Monday.

Reply:
This makes sense, this is a nice improvement